shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/MigrationJobAPI.java

```diff
@@ -45,9 +46,11 @@ public interface MigrationJobAPI extends PipelineJobAPI, MigrationJobPublicAPI,
      * @param jobConfig job configuration
      * @param jobShardingItem job sharding item
      * @param pipelineProcessConfig pipeline process configuration
+     * @param writeRateLimitAlgorithm write rate limit algorithm
      * @return task configuration
      */
-    TaskConfiguration buildTaskConfiguration(MigrationJobConfiguration jobConfig, int jobShardingItem, PipelineProcessConfiguration pipelineProcessConfig);
+    TaskConfiguration buildTaskConfiguration(MigrationJobConfiguration jobConfig, int jobShardingItem, PipelineProcessConfiguration pipelineProcessConfig,
+                                             JobRateLimitAlgorithm writeRateLimitAlgorithm);
```

Comment:
Looks it's not necessary to add new parameter `JobRateLimitAlgorithm writeRateLimitAlgorithm` for `buildTaskConfiguration` method, since AbstractPipelineProcessContext's constructor `AbstractPipelineProcessContext(final String jobId, final PipelineProcessConfiguration originalProcessConfig)`, we could just build a PipelineProcessContext instance to get writeRateLimitAlgorithm (need to add get writeRateLimitAlgorithm method in PipelineProcessContext interface).